### PR TITLE
change whatsapp message

### PR DESCRIPTION
**Pull request by kalapathar**
"CALL TO ALL US CITIZENS: This is how we defeat the corona virus: together we make the invisible virus visible."

will change to 

"CALL TO ALL Nepali Citizens: This is how we defeat the corona virus: together we make the invisible virus visible."

### CODE REVIEW

File: app/views/partials/share.ejs [hunk 1]
```diff
@@ -2,11 +2,10 @@
 <!--  -->
 <% const twitterUrl = htmlLang === 'nl' ? 'onzecorona.nl' : baseUrl %>
 <!--  -->
-<% const encode = encodeURIComponent; const message = encode(__(`CALL TO ALL US
-CITIZENS: This is how we defeat the corona virus: together we make the invisible
+<% const encode = encodeURIComponent; const message = encode(__(`CALL TO ALL Nepali: This is how we defeat the corona virus: together we make the invisible
 virus visible.`) + " " + __(`{{ amount }} people already shared their health
 status.`, { amount }) + " " + __(`Fill out this form - it takes 1 minute - and
-help the United States gain control over COVID-19.`)) %>
+help Nepal gain control over COVID-19.`)) %>
```
Comment (fossecode) [inline app/views/partials/share.ejs]: This should still be `help the United States gain control over COVID-19`, and then you just change the translation in `en-NE` (not the key of the object but the value).

Comment (kalapathar) [reply app/views/partials/share.ejs]: @fossecode so I am guessing I need to update this in my config.json ```"SUPPORTED_LOCALES": ["en-NE","ne"],```. is there any other change that I need to make after I change this? Do I have to run `deploy-prod.sh` after that?

**Review by fossecode — APPROVED** [review]

File: app/views/partials/share.ejs (hunk 1 above)
Comment (fossecode) [reply app/views/partials/share.ejs]: Yes correct, just change the config and then run `deploy-prod.sh`

**Comment by kalapathar** [comment]
@fossecode I am just testing my changes here: 
https://dev.coronastatusnp.com/

my config.json looks like this:```{
  "BASE_URL": "coronastatusnp.com",
  "COUNTRY": "Nepal",
  "COUNTRY_CODE": "np",
  "DB_PATH": "./covid_db",
  "LOCALE": "en-NE",
  "MAP_CENTER": "85.2911132, 27.7090319",
  "MAP_MAX_ZOOM": 13,
  "MAP_ZOOM": 4,
  "PASSCODE_LENGTH": 4,
  "REDIRECT_TO_GOVERNMENT": false,
  "SUPPORTED_LOCALES": ["en-NE","ne"],
  "THOUSAND_SEPARATOR": " ",
  "TWITTER": "coronastatusNO",
  "ZIP_GUIDE": true,
  "ZIP_PATTERN": "[A-Za-z0-9]{5}",
  "ZIP_PLACEHOLDER": "44600"
}``` but I don't see the other language dropdown at all now.

**Pull request merged**